ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -861,6 +861,42 @@
               recipients: projectantrea-dev@googlegroups.com rahulj@vmware.com
           triggers: []
           wrappers: []
+      - 'cloud-{name}-{test_name}':
+          test_name: auto-scaling-agent
+          node: antrea-cloud
+          description: 'This is cloud job of auto scaling agents for antrea.'
+          branches:
+          - '${{ANTREA_GIT_REVISION}}'
+          repo_url: '${{ANTREA_REPO}}'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              # Prevent exposing credentials in the console output by adding set +x
+              # This is to avoid other developers removing this set+x by accident
```

Comment:
If the credentials are injected via the credentials-binding below, the variable values in the console log are shadowed by Jenkins automatically. Maybe no need to use `set +x` here.